common/redis/redis.go

```diff
@@ -0,0 +1,43 @@
+package redis
+
+import (
+	"errors"
+	"github.com/go-redis/redis"
+	"github.com/spf13/viper"
+)
+
+var redisClient redis.UniversalClient
+
+func Init() error {
```

Comment:
궁금!
[go init function](https://medium.com/golangspec/init-functions-in-go-eac191b3860a) 을 사용하지 않고 모듈 외부에서 초기화를 요청하도록 처리한 이유가 있을까여?

Reply:
- [NewUniversalClient](https://godoc.org/github.com/go-redis/redis#NewUniversalClient) 초기화시 addr을 여러개 입력하면 cluster 모드로 사용합니다.

> 1. if a MasterName is passed a sentinel-backed FailoverClient will be returned 
> 2. if the number of Addrs is two or more, a ClusterClient will be returned 
> 3. otherwise, a single-node redis Client will be returned.

- route.go에서 서버 로딩시에 초기화 하도록 하고싶었는데 go스럽지 못한 방법이었던것 같네요. ㅋㅋ init function & route.go에 "import _ github.com/itslaves/rentalgames-server/common/redis"로 초기화 하는게 더 좋은방법같아서 수정해놓을게요~

Reply:
@freddieyyh 
오 그렇군여! 굿굿
NewUniversalClient 옵션 사용법(?)을 `config/README.md` 이쪽에도 같이 넣어두면 나중에 설정 작업할 때 도움이 될듯 합니다.